pkgs/development/python-modules/openai/default.nix

```diff
@@ -18,9 +20,18 @@
 , wandb
 }:
 
+let
+  datalib = [
+    numpy
+    openpyxl
+    pandas
+    pandas-stubs
+  ];
```

Comment:
Please inline this and use rec

Reply:
Alrighty. (I thought it was best practice not to use `rec` if it was possible to do so without too much trouble.)